2200034/half.py

```diff
@@ -0,0 +1,22 @@
+TC = int(input())
+cnt = 0
+for i in range(1, TC + 1):
+    S = list(input())
+
+    for a in range(4):
+
+        if S.count(S[a]) == 2:
+            cnt += 1
+            if cnt == 4:
+                print(f'#{i} Yes')
+                cnt = 0
+
+        elif S.count(S[a]) != 2:
+            print(f'#{i} No')
+            cnt = 0
+            break
```

Comment:
깔끔한 풀이네요! 👍
```suggestion
        if S.count(S[a]) == 2:
            cnt += 1
        if cnt == 4:
            print(f'#{i} Yes')
            cnt = 0

        elif S.count(S[a]) != 2:
            print(f'#{i} No')
            cnt = 0
            break
```
파이썬을 작성하실 땐 여백에 신경 써주세요!